iothub_client/tests/blob_ut/blob_ut.c

```diff
@@ -1252,4 +1254,24 @@ TEST_FUNCTION(Blob_UploadMultipleBlocksFromSasUri_returns_BLOB_ABORTED_when_call
     gballoc_free(fakeContext.fakeData);
 }
 
+TEST_FUNCTION(Blob_UploadMultipleBlocksFromSasUri_with_empty_payload)
+{
+    ///arrange
+    BLOB_UPLOAD_CONTEXT_FAKE fakeContext;
+    fakeContext.blockSent = 0;
+    fakeContext.blockSize = 1;
+    fakeContext.blocksCount = 0;
+    fakeContext.fakeData = NULL;
+    fakeContext.abortOnBlockNumber = 5;
+
+    ///act
+    BLOB_RESULT result = Blob_UploadMultipleBlocksFromSasUri("https://h.h/something?a=b", FileUpload_GetFakeData_Callback, &fakeContext, &httpResponse, testValidBufferHandle, NULL, NULL, NULL);
+
+    ///assert
+    ASSERT_ARE_EQUAL(BLOB_RESULT, BLOB_OK, result);
```

Comment:
I get this is a pre-existing condition in the other tests, but... couldn't we 

```c
ASSERT_ARE_EQUAL(int, 200, httpResponse);
```

Also I'd never noticed before but `httpResponse` being a global seems weird should be just on stack per each fcn?  Maybe more concerning to keep it non-global if(?) you start ASSERT on its value in test.